cal/__init__.py

```diff
@@ -27,8 +28,19 @@ def page_not_found(e):
 
 @app.route('/')
 def home():
-    events = Event.query.all()
-    return render_template('index.html', events=events)
+	events = Event.query.order_by(Event.start).all()
+	textsearch = Event.query.whoosh_search('Lerner').all()
+	print textsearch
```

Comment:
I would get rid of the `print` statement and the `textsearch`. As long as we know it works, we can get rid of testing stuff.